### PR TITLE
Fix Travis errors

### PR DESCRIPTION
**Pull request by pedrobaeza**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Ready to review.

After hard-trying lot of things, the only solution I find is to exclude the conflictive module from the joined test. In the middle of the road, I have cleaned some things to avoid possible future conflicts.

**Comment by hbrunn** [comment]
I see some changes (removal of partner class, xml changes) that are not pep8 related. Could you explain them please?

**Comment by pedrobaeza** [comment]
Yeah, the title is Fix Travis (not only PEP8), and I have take the ocassion to improve some code. You can see separated commits for them to be easily recognised.

**Comment by gurneyalex** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,8 @@ python:
 
 env:
   - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0" EXCLUDE="portal_partner_merge"
+  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0" EXCLUDE="portal_partner_merge"
```
Comment (eLBati) [inline .travis.yml:9]: Doesn't `portal_partner_merge` work alone neither? I mean with `INCLUDE="portal_partner_merge"`

Comment (pedrobaeza) [inline .travis.yml:9]: Yeah, I suppose. I didn't try. Let me add it.

Comment (pedrobaeza) [inline .travis.yml:9]: Done. Let's see the result.

**Comment by StefanRijnhart** [comment]
Looks good, thanks! :+1:

**Pull request merged**